web-app/CKS.Web.Test/IntegrationTests/Infrastructure/CKSWebApplicationFactory.cs

```diff
@@ -19,17 +19,30 @@ public class CKSWebApplicationFactory : WebApplicationFactory<Startup>
 	{
 		private readonly ITestOutputHelper _output;
 		private readonly IList<Action<IServiceCollection>> serviceCollectionsActions = new List<Action<IServiceCollection>>();
+		private string webRootPath = null;
 
 		public CKSWebApplicationFactory(ITestOutputHelper output)
 		{
 			_output = output;
 			_output.WriteLine("CKSWebApplicationFactory.constructor");
 		}
 
+		protected override IWebHostBuilder CreateWebHostBuilder() => base.CreateWebHostBuilder();
```

Comment:
We don't need this line if we're not calling `UseEnvironment` here. I'd rather we do:
```suggestion
		protected override IWebHostBuilder CreateWebHostBuilder() =>
    base.CreateWebHostBuilder().UseEnvironment("Testing");
```
(and remove line 36) as [that's what the docs say](https://docs.microsoft.com/en-us/aspnet/core/test/integration-tests?view=aspnetcore-3.1#set-the-environment).